### ISSUE TITLE
Champion search resets when any player locks in

### ISSUE THREAD

**Issue by Erj4** [OrangeNote/RuneBook#75]
While searching for a champion to get rune pages for (before locking in), the search is cleared as well as any results whenever any player locks in. Only happens when auto select is enabled.

**Comment by jeske** [comment]
Yeah... auto-select should never "clear" the champion selected....
